examples/creatives/banner_nonlinear.html

```diff
@@ -0,0 +1,57 @@
+<html>
+  <head>
+    <script src="../simid_protocol.js"> </script>
+    <script src="base_simid_creative.js"> </script>
+    <script src="banner_nonlinear.js"> </script>
+  </head>
+  <style>
+    #close_ad {
+      color: red;
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    #minimize_ad {
+      color:rgb(78, 78, 78);
+      border: solid white 1px;
+      font-size: 14;
+      float: right;
+    }
+    button {
+      cursor: pointer;
+    }
+
+    body {
+      background: rgba(0,0,0,0.5);
+      width: auto;
+      text-align: center;
+      transition: opacity .5s cubic-bezier(0.0,0.0,0.2,1);
+      text-shadow: 0 0 4px rgba(0,0,0,0.75);
+      cursor: pointer;
+    }
+
+    body:hover {
+      background: rgba(0,0,0,0.7);
+    }
+
+    #ad_text {
+      color: white;
+      text-align: center;
+    }
+
+  </style>
+  <body>
+    <button id="minimize_ad">_</button>
```

Comment:
maybe try a hyphen here instead of an underscore

Reply:
The hyphen looks a little small and weird (more like a subtraction button than a minimize button), the underscore looks a bit more consistent with the minimize button in web browsers